src/plugin-hooks/useRowSelect.js

```diff
@@ -96,6 +95,8 @@ function reducer(state, action, previousState, instance) {
   if (action.type === actions.init) {
     return {
       selectedRowIds: {},
+      lastSelectedRowId: null,
+      lastShiftSelectedRowId: null,
```

Comment:
`lastShiftSelectedRowId` can be removed..

Reply:
Oops. Also I wasn't sure what the convention would be here. Should they still be initialized or what about leaving it undefined until first set?